config/locales/interface/slides/en_data.yml

```diff
@@ -169,8 +169,13 @@ en:
     data_export_hydrogen:
       title: Hourly curves for hydrogen
       short_description:
-      description: |-
-        Below you can download information about hydrogen demand, supply and storage per hour in your scenario.
+      description: |
+        Below you can download two exports concering hydrogen.
+        <br/><br/>
+        The "Hydrogen demand, production and storage per hour" export contains information about hydrogen demand, supply and storage per hour in your scenario.
+        <br/><br/>
+        The "Hydrogen integral costs" export contains the integral costs of hydrogen production, the production of hydrogen, 
+        the production cost per Mwh for each generation method and the production curve of hydrogen for each generation method on an hourly basis.
```

Comment:
I would denote MWh instead of Mwh and maybe link the new hydrogen costs chart in this description or set the chart as the default chart for this slide.